### PR TITLE
Delete unnecessary words

### PR DESCRIPTION
**Pull request by michaelmhoffman**
Delete words where this can be done with minimal change in meaning.

### CODE REVIEW

File: sections/05_treat.md [hunk 1]
```diff
@@ -375,15 +375,9 @@ atom-atom interactions and the forces that govern binding affinity.
 
 #### *De novo* drug design
 
-Whereas the goal of virtual screening is to find active molecules by predicting
```
Comment (cgreene) [inline sections/05_treat.md]: @agitter tagging you on this.

File: sections/05_treat.md [hunk 2]
```diff
@@ -138,12 +138,12 @@ Wang et al. [@doi:10.1093/bioinformatics/btt234] trained individual RBMs for
 each target in a drug-target interaction network and used these models to
 predict novel interactions pointing to new indications for existing drugs. Wen
 et al. [@doi:10.1021/acs.jproteome.6b00618] extended this concept to deep
-learning by creating a DBN called DeepDTIs, which is able to predict
-interactions on the basis of chemical structure and protein sequence features.
+learning by creating a DBN called DeepDTIs, which predicts
+interactions using chemical structure and protein sequence features.
 
-Drug repositioning appears to be an obvious candidate for deep learning both
+Drug repositioning appears an obvious candidate for deep learning both
```
Comment (agitter) [inline sections/05_treat.md]: Perhaps `is an` instead of `appears an`?  Or leave it as is if you prefer.

File: sections/05_treat.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ repositioning via deep learning is currently missing.
 
 #### Ligand-based prediction of bioactivity
 
-In the biomedical domain, high-throughput chemical screening aims to improve
+High-throughput chemical screening aims to improve
```
Comment (agitter) [inline sections/05_treat.md]: I'd argue we need to keep `In the biomedical domain,`.  That may be implied by the review topic, but there is an entirely different line of high-throughput chemical screening in field such as materials science that has its own meaning.

File: sections/05_treat.md (hunk 1 above)
Comment (agitter) [inline sections/05_treat.md]: `doi:10.1021/acs.jmedchem.5b01849` was a good reference so I prefer to not cut it.  I'm okay with the reduced text.

@mrwns wrote some of this as well.

File: sections/04_study.md [hunk 4]
```diff
@@ -663,9 +663,8 @@ only thousands of full-sequenced genomes as compared to hundreds of thousands of
 16S rRNA sequences available for training.
 
 However, because RNNs have been applied to base calls for the Oxford Nanopore
-long-read sequencer with some success [@tag:Boza] (discussed further in the next
-section), one day the entire pipeline, from denoising of through functional
-classification, may be combined into one step by using powerful LSTMs
+long-read sequencer with some success [@tag:Boza] (discussed below), one day the entire pipeline, from denoising of through functional
```
Comment (michaelmhoffman) [inline sections/04_study.md]: Fixed in michaelmhoffman/deep-review@5c4747997f16ec1e7e156c230babadd1a3f214be

File: sections/05_treat.md (hunk 2 above)
Comment (michaelmhoffman) [inline sections/05_treat.md]: Leaving as is to avoid "to be" verb

File: sections/05_treat.md (hunk 3 above)
Comment (michaelmhoffman) [inline sections/05_treat.md]: I found the previous phrasing awkward and moves the true subject away from the topic position, but I see your point. Changed to "screening in biomedical research" in michaelmhoffman/deep-review@c82e777e96597381100ab0551a24e27bfdfc36c6.

File: sections/05_treat.md (hunk 1 above)
Comment (michaelmhoffman) [inline sections/05_treat.md]: Moved the citation to the subsequent paragraph in michaelmhoffman/deep-review@904b89b9c847b90454e23ae690b99273ad749957

**Comment by michaelmhoffman** [comment]
@agitter Ready for your review.

**Pull request merged**